src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -13,19 +13,22 @@
     public static final String MESSAGE_INVALID_PATIENT_DISPLAYED_INDEX = "The patient index provided is invalid";
     public static final String MESSAGE_FORCE_DELETE_PATIENT_REQUIRED =
             "The patient has existing appointments in appointment schedule."
-            + " Force delete command format required! \n%1$s";
-    public static final String MESSAGE_CLEAR_APPOINTMENTS_REQUIRED =
+                    + " Force delete command format required! \n%1$s";
+    public static final String MESSAGE_CLEAR_APPOINTMENTS_BEFORE_PATIENTS_REQUIRED =
             "Patient records cannot be cleared until appointment schedule is cleared.\n"
-            + "Use the command \"clear-appt\" to clear the appointment schedule.";
-    public static final String MESSAGE_DELETE_PATIENT_SUCCESS = "Deleted Person: %1$s";
+                    + "Use the command \"clear-appt\" to clear the appointment schedule.";
+    public static final String MESSAGE_DELETE_PATIENT_SUCCESS = "Deleted Patient: %1$s";
 
-    //=================================Doctor related messages===============================================
+    //==================================Doctor related messages===============================================
```

Comment:
this extra `=` misaligns it 😢

Reply:
Meant to align the tail of the comment, will commit the changes.